page-changelog.php

```diff
@@ -15,6 +15,108 @@
 </ul>
 
 <a name="latest10.0"></a></li>
+<h3>Version 10.0.4 RC2<small>Nov 22 2017</small></h3>
+<p>Release notes: <a  href="https://doc.owncloud.org/server/10.0/admin_manual/release_notes.html#changes-in-10-0-3">Release notes</a></p>
```

Comment:
Hmm I see now that this link says 10-0-3. Change it to 10-0-4. The section isn't ready yet, I believe the PR is still open.